OJP/OJP_Trips.xsd

```diff
@@ -604,24 +604,24 @@
 	</xs:complexType>
 	<xs:complexType name="PathGuidanceStructure">
 		<xs:annotation>
-			<xs:documentation> description of a piece of a TRIP. May include geographic information, turn instructions and accessibility information </xs:documentation>
+			<xs:documentation> description of a piece of a TRIP. May include geographic information, turn instructions and accessibility information.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element name="PathGuidanceSection" type="PathGuidanceSectionStructure" maxOccurs="unbounded">
 				<xs:annotation>
-					<xs:documentation>one or more path guidance sections that build the trip Leg</xs:documentation>
+					<xs:documentation>One or more PATH GUIDANCE sections that build the LEG. The sections may need to be rather short to incororpate the necessary guidance in a detailed level. This may depend a lot on the MODE and on the type of guidance that is needed.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
 	<xs:complexType name="PathGuidanceSectionStructure">
 		<xs:annotation>
-			<xs:documentation>[an extended definition of a NAVIGATION PATH in TMv6 to include the textual navigation instructions] description of a piece of a TRIP. May include geographic information, turn instructions and accessibility information </xs:documentation>
+			<xs:documentation>An extended definition of a NAVIGATION PATH in TMv6 and PATH GUIDANCE to include the textual navigation instructions] description of a piece of a TRIP. May include geographic information, turn instructions and accessibility information.</xs:documentation>
```

Comment:
Leftover closing bracket as `instructions]`. Once again "piece", for a trip/journey I only know "part"..

Reply:
changed